src/mutations/appData.ts

```diff
@@ -62,6 +74,7 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
     },
     onSuccess: (newAppData) => {
       queryConfig?.notifier?.({ type: patchAppDataRoutine.SUCCESS, payload: newAppData });
+      // TODO: Implement local patching mecanism
```

Comment:
TODO ;)

Reply:
I removed it because actually, the patch already happens in the `onMutate` callback. However, it makes me wonder if we shouldn't move this to the `onSuccess` so that patch events are locally persisted only if the request is completed successfully.